sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/LastEnqueuedEventProperties.java

```diff
@@ -30,11 +31,41 @@ public class LastEnqueuedEventProperties {
      *     has not been retrieved, yet.
      */
     public LastEnqueuedEventProperties(Long lastSequenceNumber, Long lastOffset, Instant lastEnqueuedTime,
```

Comment:
Is it intentional that we don't deprecate this Ctr overload (with long offset)?

Reply:
Ahh. I missed the deprecation on the constuctor. Thanks!